modules/migration-restic-errors.adoc

```diff
@@ -95,7 +95,7 @@ The output identifies the Restic Pod that logged the errors:
   completionTimestamp: 2020-05-01T20:49:12Z
   errors: 1
   resticErrors: 1
-...
+  [...]
```

Comment:
I've never seen us indicate an optional(?) part of a spec this way.

@openshift/team-documentation - Do we have any thoughts about having `[...]` in YAML specs?

Reply:
It's an [ellipsis](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc), not an option.